src/formula/callable_objects.hpp

```diff
@@ -25,10 +25,31 @@ class team;
 class terrain_type;
 class unit;
 class unit_type;
+namespace game_events {class queued_event;}
 
 namespace wfl
 {
 
+class gamestate_callable : public formula_callable
+{
+public:
+	// TODO: Constructor
```

Comment:
Without a constructor, is there any data that's accessible via this object?

Could the `world`-related bits be removed from the current PR, and added as a separate change later?

Reply:
It's accessing all its data through the `resources` namespace, so yeah, there's data accessible via this object.